flink-formats/flink-orc/src/test/java/org/apache/flink/orc/OrcColumnarRowSplitReaderTest.java

```diff
@@ -283,8 +284,12 @@ public void testReadFileWithTypes() throws IOException {
 			for (int i = 0; i < rowSize - 1; i++) {
 				col0.vector[i] = i;
 				col1.vector[i] = i;
-				col2.time[i] = i * 1000;
-				col2.nanos[i] = i;
+
+				Timestamp timestamp = Timestamp.valueOf(
+						padZero(4, i + 1000) + "-01-01 00:00:00." + i);
```

Comment:
The test would cover more cases if the values are more representative (rather than a lot of zeros for parts of the timestamp).

Reply:
Actually it has test `1000-00-00` until `2023-00-00`. But I can assign all values too.